ansible/roles/oracle-19c/README.md

```diff
@@ -0,0 +1,23 @@
+This role installs Oracle 11g. It assumes the installation disks `\u01` and `\u02` have already been setup (with `disks` role for example).
```

Comment:
this is the 11g readme, probably outdated or at least not trusted for this role. So we could just have an empty readme for now?